app/products/calls/connection/websocket_client.ts

```diff
@@ -143,40 +143,39 @@ export class WebSocketClient extends EventEmitter {
     }
 
     close() {
-        if (this.ws) {
-            this.closed = true;
-            this.ws.close();
-            this.ws = null;
-            this.seqNo = 1;
-            this.serverSeqNo = 0;
-            this.connID = '';
-            this.originalConnID = '';
-        } else {
-            this.emit('close');
+        this.closed = true;
+        this.ws?.close();
+        this.ws = null;
+        this.seqNo = 1;
+        this.serverSeqNo = 0;
+        this.connID = '';
+        this.originalConnID = '';
+    }
 
-            const now = Date.now();
-            if (this.lastDisconnect === 0) {
-                this.lastDisconnect = now;
-            }
+    reconnect() {
+        const now = Date.now();
+        if (this.lastDisconnect === 0) {
+            this.lastDisconnect = now;
+        }
 
-            if ((now - this.lastDisconnect) >= wsReconnectionTimeout) {
-                this.closed = true;
-                this.emit('error', wsReconnectionTimeoutErr);
-                return;
-            }
+        if ((now - this.lastDisconnect) >= wsReconnectionTimeout) {
+            this.closed = true;
+            this.emit('error', wsReconnectionTimeoutErr);
+            return;
+        }
 
-            setTimeout(() => {
-                if (!this.ws && !this.closed) {
-                    this.init(true);
-                }
-            }, this.reconnectRetryTime);
+        setTimeout(() => {
+            if (!this.closed) {
+                logDebug('reconnecting');
```

Comment:
maybe make this message a bit more verbose, to understand the actual log.. `reconnecting calls websocket {url}` or something similar

Reply:
Sure, I forget there's no automatic trace in mobile logs. Otherwise the location would be taken care of.

Reply:
thanks